mobilizon_bots/publishers/abstract.py

```diff
@@ -0,0 +1,78 @@
+import inspect
+import logging
+
+from abc import ABC, abstractmethod
+
+logging.basicConfig(
```

Comment:
Io il logging lo configurerei in un solo posto, insieme alla configurazione del db. Anche perche' loggerei sia su stdout che su file. Cosa dite?

Reply:
sì, assolutamente.